README.md

```diff
@@ -6,7 +6,7 @@ Geospatial Research Institute Toi Hanagarau. This application is used to visuali
 ![image](https://user-images.githubusercontent.com/41398636/126084844-1bad1906-8784-4f9b-93ec-df0e5e995dd6.png)
 
 
```

Comment:
I think this is getting close to where you might want to split it out into your wiki (or other documentation). I wonder about having two sections in your wiki - package usage (the first 4/5 of your README.md), and package developers (the contents already in the wiki + the contributions, license section of your README.md)

Reply:
I agree, I've added a documentation issue #21 to make sure I get around to this before working on the next part of the project

Reply:
Can be done in a separate issue.